02Lesson/01Project/index.html

```diff
@@ -1,9 +1,44 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
-</head>
-<body>
+  <head>
+    <title>
+      Arnold Wells Photography
+    </title>
+    <meta charset="UTF-8">
+  </head>
+  <body style="font-family: Arial">
+
+    <img src="https://scontent-dft4-2.xx.fbcdn.net/v/t1.0-9/10930907_905943542773873_7473205960492472388_n.jpg?oh=c1863cdb875abfe3f45b05069125126e&oe=589D2EAE" alt="Smiley face" width="500"/>
+    <hr>
+  <h1>
+    Arnold Is An Awesome Photographer!
+  </h1>
+
+  <p>
+    Arnold provides entertainment and editorial photographic services for all
+    types of publications and corporate events.  Arnold Will always get the shot!
+
+    Published
+  </p>
+    <ul>
```

Comment:
Indentation: After closing tag, the next opening tag should be on the same level.